models.py

```diff
@@ -0,0 +1,44 @@
+from typing import NamedTuple, List
+
+import numpy as np
+
+
+class CoordinatesRange(object):
+    def __init__(self, vector: np.ndarray):
+        """
+        :param vector: vector with locations across single coordinate
+        """
+        self.Min = int(vector.ravel().min())
+        self.Max = int(vector.ravel().max())
+
+
+class DatasetMetadata(object):
+    def __init__(self, dataset_id: int, name: str, spectra_number: int,
+                 x_range: CoordinatesRange, y_range: CoordinatesRange):
+        """
+        :param name: name of dataset to extract metadata from
+        """
+        self.Id = dataset_id
+        self.Name = name
+        self.SpectraNumber = spectra_number
+        self.XRange = vars(x_range)
+        self.YRange = vars(y_range)
+
+
+DivikResult = NamedTuple('DivikResult', [
```

Comment:
It will be better to parse data in front-end to plotly if API send the data from DivikResult in format from function toHeatmap (https://github.com/spectre-team/spectre/blob/develop/src/app/heatmaps/shared/heatmap-util.ts)

Reply:
Ok, accepted. I'll update it.